fhir-model/src/main/java/com/ibm/fhir/model/lang/util/LanguageRegistry.java

```diff
@@ -0,0 +1,9012 @@
+/*
```

Comment:
I went back and forth on this, and since the contents of IANA registry will be static, I leaned toward avoiding reading from a file during runtime to just make it as fast as possible, plus it allowed me to look at the generated lists/maps to confirm nothing was missing during the parsing. This intermediate step could certainly be removed, and just parse the file directly each time at runtime, but since I have it generated, I felt leaving it that way was fine.

Reply:
I'd rather parse ahead of time - the reason is the first request that activates the code path is truly going to be delayed.  Loading this up ahead of time and populating the maps ensures when the context is finished activating the app is truly ready to service a request.

Reply:
The parsing of file would still happen at startup via FHIRServletContextListener, I think, so does that change your opinion?

Reply:
Then I agree with Mike. I think that makes sense